src/features/commands/opencode-command.ts

```diff
@@ -0,0 +1,87 @@
+import { basename, join } from "node:path";
+import { AiFileParams, ValidationResult } from "../../types/ai-file.js";
+import { readFileContent } from "../../utils/file.js";
+import { parseFrontmatter } from "../../utils/frontmatter.js";
+import { RulesyncCommand, RulesyncCommandFrontmatter } from "./rulesync-command.js";
+import {
+  ToolCommand,
+  ToolCommandFromFileParams,
+  ToolCommandFromRulesyncCommandParams,
+  ToolCommandSettablePaths,
+} from "./tool-command.js";
+
+export type OpencodeCommandParams = AiFileParams;
+
+export class OpencodeCommand extends ToolCommand {
+  static getSettablePaths(_options: { global?: boolean } = {}): ToolCommandSettablePaths {
+    return {
+      relativeDirPath: join(".opencode", "commands"),
```

Comment:
```suggestion
      relativeDirPath: join(".opencode", "command"),
```

Referece to foloowing document.

https://opencode.ai/docs/commands/

> Create markdown files in the command/ directory to define custom commands.